### ISSUE TITLE
📢 IMPORTANT: Submit PRs Only After Raising and Being Assigned to Issues

### ISSUE THREAD

**Issue by aslams2020** [aslams2020/Stark-Tech-Portfolio#64]
### Discussed in https://github.com/aslams2020/Stark-Tech-Portfolio/discussions/61

<div type='discussions-op-text'>

<sup>Originally posted by **aslams2020** October  2, 2024</sup>
Hi everyone,

I’ve noticed some contributors are **submitting pull requests (PRs) without first raising an issue or getting assigned to one.** 
To keep things organized and in line with the **project guidelines.** 

Read How To Contribute and Guidelines:
[Click Here to View](https://github.com/orgs/GSSoC24/discussions/175)

### I want to remind everyone of the following steps:
- **Raise an Issue**: Before working on any changes, make sure you create an issue describing the problem or feature you want to address.

- **Get Assigned:** Wait for the project admins (PAs) to assign the issue to you. Only after you’ve been assigned should you start working on a pull request.

- **Link the Issue in Your PR:** Once assigned, include the issue number in your pull request description for easy reference.

Submitting a PR without following these steps may lead to your work being put on hold, especially if someone else is already assigned to the issue. 
**We want to avoid duplication and ensure fair contribution opportunities for everyone.**

If you’ve submitted a PR without raising or being assigned to an issue, please create an issue now, and we’ll proceed accordingly.

**Thank you for your understanding and cooperation!**</div>

**Comment by sanjanasp786** [comment]
i would like to work on this issue..please assign this to me

**Comment by aslams2020** [comment]
This is not an issue, Please Read it Carefully.
